cluster-http/src/test/scala/akka/cluster/http/management/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -275,5 +286,91 @@ class ClusterHttpManagementRoutesSpec
         }
       }
     }
+
+    "return shard region details" when {
+
+      "calling GET /shard_regions/{name}" in {
+        val config = ConfigFactory.parseString(
+          """
+            |akka.cluster {
+            |  auto-down-unreachable-after = 0s
+            |  periodic-tasks-initial-delay = 120 seconds // turn off scheduled tasks
+            |  publish-stats-interval = 0 s # always, when it happens
+            |  failure-detector.implementation-class = akka.cluster.FailureDetectorPuppet
+            |  sharding.state-store-mode = ddata
+            |}
+            |akka.actor.provider = "cluster"
+            |akka.remote.log-remote-lifecycle-events = off
+            |akka.remote.netty.tcp.port = 0
+          """.stripMargin
+        )
+        val configClusterHttpManager = ConfigFactory.parseString(
+          """
+            |akka.cluster.http.management.hostname = "127.0.0.1"
+            |akka.cluster.http.management.port = 20100
```

Comment:
Ah yes, and that works because `Net.serverSocket` will set `SO_REUSEADDR` when binding for real. Not sure I expected that, but turns out nicely here :). (https://eklitzke.org/binding-on-port-zero)